srcpkgs/source-sans-pro/template

```diff
@@ -1,17 +1,16 @@
 # Template file for 'source-sans-pro'
 pkgname=source-sans-pro
-version=3.006
+version=3.028
 revision=1
-_relver="3.006R"
-archs=noarch
+_relver="3.028R"
 wrksrc="source-sans-pro-${_relver/\//-}"
```

Comment:
Why don't we use `wrksrc="source-sans-pro-${version}R"` instead?

Reply:
No idea :P

Reply:
`_relver` wasn't similar to `version` few version ago, let's remove it now.

Reply:
Done, thanks @sgn @Chocimier !